inspirehep/modules/records/serializers/schemas/json.py

```diff
@@ -44,8 +45,17 @@ class RecordSchemaJSONUIV1(Schema):
 
 class MetadataAuthorsSchemaV1(Schema):
 
-    authors = fields.Raw(dump_only=True)
+    authors = fields.Method('get_authors', default=[])
     collaborations = fields.Raw(dump_only=True)
+    supervisors = fields.Method('get_supervisors', default=[])
+
+    def get_authors(self, data):
+        record = LiteratureRecord(data)
```

Comment:
I just realised ``LiteratureRecord`` does an ES call each time, can we move it this somewhere else?